### ISSUE TITLE
AttributeError for NoneType makes any video playback on Netflix impossible

### ISSUE THREAD

**Issue by YamiNoSenshi** [CastagnaIT/plugin.video.netflix#1069]
#### Your Environment
- Netflix add-on version: 1.12.2
- Operating system version/name: LibreElec 9.2.6 - Kodi 18.9
- Device model:  Raspberry Pi 4B

Used Operating system:
* [ ] Android
* [ ] iOS
* [ ] Linux
* [ ] OSX
* [X] Raspberry-Pi
* [ ] Windows

### Describe the bug

Trying to play any video results in the following error "AttributeError: 'NoneType' object has no attribute 'get'". Playback worked fine a week ago. No changes were made to the system and suddenly every video results in this error.

#### Expected behavior

Video playback should start.

#### Actual behavior

Video playback fails to start and displays the following error: "AttributeError: 'NoneType' object has no attribute 'get'".

#### Steps to reproduce the behavior

1. Open my custom shortcut with the following action to move directly to my own profile in the account: ActivateWindow(Videos,"plugin://plugin.video.netflix/directory/home/?switch_profile_guid=IVVEGDS7CZFQDBTC4E6KFUWXGQ",return) (Note: Going through the add-on and manually selecting the profile has the same result.)
2. Navigate to any video in Netflix.
3. Try to play the video
4. Video does not play and an error pop-up is shown.

#### Possible fix

Judging from the log file and the error message I'm expecting the plugin is running into a null value and crashing while calling the get action on it.

### Debug log

The debug log can be found here:

[01_KODI.log](https://github.com/CastagnaIT/plugin.video.netflix/files/5978669/01_KODI.log)

#### Other information

When trying to find a solution the only issue that appeared somewhat related was this issue about the update action: [link](https://github.com/im85288/service.upnext/issues/176).

#### Screenshots

![IMG_6699](https://user-images.githubusercontent.com/6615747/107891391-a7cf1300-6f1e-11eb-8937-c9229f5a47af.JPG)

**Comment by jakermx** [comment]
I suggest you to read this

https://github.com/CastagnaIT/plugin.video.netflix/wiki/VPN-and-Proxy

NF Addon uses an internal proxy for any request, asi it has the same ID as the VPN Peer, it will faill to read the response...try it without VPN and it will work as supposed to

**Comment by CastagnaIT** [comment]
please _completely_ disable the vpn add-ons from kodi settings,
some vpn addons cause problems also if are set as disabled from his addon settings

**Comment by YamiNoSenshi** [comment]
As you mentioned the VPN addon was only disabled in it's own configuration settings (already had issues with it).
I have disabled it completely from the kodi addon settings now and rerun the test.

Unfortunately I'm still getting the same error.
Here is the new log file: 
[01_KODI.log](https://github.com/CastagnaIT/plugin.video.netflix/files/5980345/01_KODI.log)

**Comment by CastagnaIT** [comment]
Now i need you update with this: [plugin.video.netflix_1.12.2_20210215.zip](https://github.com/CastagnaIT/plugin.video.netflix/files/5983465/plugin.video.netflix_1.12.2_20210215.zip)
and provide a new log to be analized thanks

**Comment by YamiNoSenshi** [comment]
Thanks, I have installed the new version and rerun the test.
Here is the new log:
[01_KODI.log](https://github.com/CastagnaIT/plugin.video.netflix/files/5983960/01_KODI.log)

**Comment by CastagnaIT** [comment]
This update should show correctly the error: [plugin.video.netflix_1.12.2_20210216_t1.zip](https://github.com/CastagnaIT/plugin.video.netflix/files/5987065/plugin.video.netflix_1.12.2_20210216_t1.zip)

but your error is due to other things, like the previously enabled VPN or deleted profile, changed account password or others account activities while the add-on was running,
i suggest you after the update to try Logout from addon settings and relogin

**Comment by YamiNoSenshi** [comment]
You were right. Logging out and reinitializing with a new Authentication key seems to have resolved the issue.
The first call to the shortcut I created resulted in a SSL error but any subsequent calls are working as expected. (even after a reboot).
I also changed the installation to the repository version again which also seems to work as expected.
Here is another log in case the information can still help development (generated with the latest version you sent):
[01_KODI.log](https://github.com/CastagnaIT/plugin.video.netflix/files/5988066/01_KODI.log)

Again, thanks for the help.
